CONTRIBUTING.md

```diff
@@ -12,7 +12,8 @@ If you want to suggest any changes to the process, please feel free to open a Pu
 There are a number of ways in which you can set up development environments for working on FDO (FIDO Device Onboarding). 
 
 ##### On Fedora host system
-In order to make a test build of this crate, when using Fedora, you can run:
+
+In order to make a test build of this crate, when using Fedora 37, 38 or 39, you can run:
```

Comment:
If fedora 36 does not work as per https://github.com/fedora-iot/fido-device-onboard-rs/issues/481, the following toolbox instruction also does not work?

https://github.com/fedora-iot/fido-device-onboard-rs/blob/be00cc172b8ebf09a510b26bba272aeeb2abfe2d/CONTRIBUTING.md?plain=1#L54-L55

I think it is better to use the latest Fedora release version and the toolbox part is also guided as "use the latest fedora release".

Reply:
Yes, that would also fail, nice catch.
Will update to say that we support the latest Fedora release and that would make everything more easier to maintain.